### ISSUE TITLE
If a CONTRIBUTING.md that is a symlink is present, the link to it should not show on new issue forms.

### ISSUE THREAD

**Issue by cirosantilli** [isaacs/github#348]
https://github.com/cirosantilli/test-symlink-contributing/issues/new

The "Please review the guidelines for contributing to this repository." should not appear in this case, just like the README.md does not appear for submodules.

Works as expected for submodules: https://github.com/cirosantilli/test-submodule-contributing/issues/new and does not show.

**Comment by stuartpb** [comment]
Alternately, if it points sensibly to a file, they should change the UI to link to the file that the symlink points to.

I'll make a revision to https://github.com/ghes/contributing-includer/blob/gh-pages/contributing.user.js to implement this.

**Comment by cirosantilli** [comment]
@stuartpb yes, that would be good as well. If implemented by GitHub, it would only require some security checks like path transversal and .git access. And if recursive symlinks are enabled, infinite symlink loops.

**Comment by cirosantilli** [comment]
GitHub implemented it silently as it does for such small features.

**Comment by stuartpb** [comment]
Did they? It looks like it's still pointing at the symlink itself for me.

**Comment by cirosantilli** [comment]
@stuartpb going nuts, nothing is changed.
